app/src/main/java/com/oneshot/rxjavaapp/feature/photolist/data/entity/PhotosPageEntity.java

```diff
@@ -0,0 +1,26 @@
+package com.oneshot.rxjavaapp.feature.photolist.data.entity;
+
+import com.oneshot.rxjavaapp.annotations.ApiSerializable;
+import com.oneshot.rxjavaapp.state.data.Result;
+
+/**
+ * Created by brettn on 9/21/16.
+ */
+@ApiSerializable
+public class PhotosPageEntity extends Result {
+
+    int current_page;
+
+
+    int total_pages;
+
+
+    int total_items;
+
+
+    PhotoEntity[] photos;
+
+    public PhotoEntity[] getPhotos() {
+        return photos;
+    }
```

Comment:
Done, remember though that these are ApiSerializable classes, so GSON handles everything for us, we only need getters for the data that we use directly